### ISSUE TITLE
Problems with Civioffice starting from Advanced Search on Wordpress

### ISSUE THREAD

**Issue by umeyermartin** [systopia/de.systopia.civioffice#70]
CiviOffice Version 1.0dev

![image](https://github.com/user-attachments/assets/09a8ef9c-a2b2-4564-b0e3-2d9fd1513f87)

PHP-module Assert.Assertion not found

How can I fix this?

**Comment by vurt2** [comment]
You have to execute "composer install" inside the extension directory. This should be noted in the documentation...

**Comment by bjendres** [comment]
> This should be noted in the documentation...

You're right, that's missing in [the documentation](https://docs.civicrm.org/civioffice/en/latest/installation/). 

@dontub or @jensschuppe Could you add that to the documentation and/or make sure it is "shipped" with the required libraries?

**Comment by peth-systopia** [comment]
@vurt2 @umeyermartin We are accepting PRs for the documentation. :-)
